region/norfair/west.json

```diff
@@ -1947,6 +1947,27 @@
                       "hits": 1
                     }}
                   ]
+                },
+                {
+                  "name": "Ice",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    "Morph",
+                    "canUseFrozenEnemies",
+                    {"heatFrames": 300}
+                  ],
+                  "note": "An ice shot through the morph tunnel can freeze the Boyon even without Wave."
+                },
+                {
+                  "name": "Heat and Lava Proof",
+                  "notable": false,
+                  "requires": [
+                    "Varia",
```

Comment:
Just a note: this would've beeen better as `h_heatProof`. We've decoupled Varia/Gravity from heatproofing via that helper.